pkg/security/utils/syscalls.go

```diff
@@ -0,0 +1,733 @@
+	{"arm64", 268}: "setns",
+	{"arm64", 269}: "sendmmsg",
+	{"arm64", 270}: "process_vm_readv",
+	{"arm64", 271}: "process_vm_writev",
+	{"arm64", 272}: "kcmp",
+	{"arm64", 273}: "finit_module",
+	{"arm64", 274}: "sched_setattr",
+	{"arm64", 275}: "sched_getattr",
+	{"arm64", 276}: "renameat2",
+	{"arm64", 277}: "seccomp",
+	{"arm64", 278}: "getrandom",
+	{"arm64", 279}: "memfd_create",
+	{"arm64", 280}: "bpf",
+	{"arm64", 281}: "execveat",
+	{"arm64", 282}: "userfaultfd",
+	{"arm64", 283}: "membarrier",
+	{"arm64", 284}: "mlock2",
+	{"arm64", 285}: "copy_file_range",
+	{"arm64", 286}: "preadv2",
+	{"arm64", 287}: "pwritev2",
+	{"arm64", 288}: "pkey_mprotect",
+	{"arm64", 289}: "pkey_alloc",
+	{"arm64", 290}: "pkey_free",
+	{"arm64", 291}: "statx",
+	{"arm64", 292}: "io_pgetevents",
+	{"arm64", 293}: "rseq",
+	{"arm64", 294}: "kexec_file_load",
+	{"arm64", 403}: "clock_gettime64",
+	{"arm64", 404}: "clock_settime64",
+	{"arm64", 405}: "clock_adjtime64",
+	{"arm64", 406}: "clock_getres_time64",
+	{"arm64", 407}: "clock_nanosleep_time64",
+	{"arm64", 408}: "timer_gettime64",
+	{"arm64", 409}: "timer_settime64",
+	{"arm64", 410}: "timerfd_gettime64",
+	{"arm64", 411}: "timerfd_settime64",
+	{"arm64", 412}: "utimensat_time64",
+	{"arm64", 413}: "pselect6_time64",
+	{"arm64", 414}: "ppoll_time64",
+	{"arm64", 416}: "io_pgetevents_time64",
+	{"arm64", 417}: "recvmmsg_time64",
+	{"arm64", 418}: "mq_timedsend_time64",
+	{"arm64", 419}: "mq_timedreceive_time64",
+	{"arm64", 420}: "semtimedop_time64",
+	{"arm64", 421}: "rt_sigtimedwait_time64",
+	{"arm64", 422}: "futex_time64",
+	{"arm64", 423}: "sched_rr_get_interval_time64",
+	{"arm64", 424}: "pidfd_send_signal",
+	{"arm64", 425}: "io_uring_setup",
+	{"arm64", 426}: "io_uring_enter",
+	{"arm64", 427}: "io_uring_register",
+	{"arm64", 428}: "open_tree",
+	{"arm64", 429}: "move_mount",
+	{"arm64", 430}: "fsopen",
+	{"arm64", 431}: "fsconfig",
+	{"arm64", 432}: "fsmount",
+	{"arm64", 433}: "fspick",
+	{"arm64", 434}: "pidfd_open",
+	{"arm64", 435}: "clone3",
+	{"arm64", 436}: "close_range",
+	{"arm64", 437}: "openat2",
+	{"arm64", 438}: "pidfd_getfd",
+	{"arm64", 439}: "faccessat2",
+	{"arm64", 440}: "process_madvise",
+	{"arm64", 441}: "epoll_pwait2",
+	{"arm64", 442}: "mount_setattr",
+	{"arm64", 443}: "quotactl_fd",
+	{"arm64", 444}: "landlock_create_ruleset",
+	{"arm64", 445}: "landlock_add_rule",
+	{"arm64", 446}: "landlock_restrict_self",
+	{"arm64", 447}: "memfd_secret",
+	{"arm64", 448}: "process_mrelease",
+	{"arm64", 449}: "futex_waitv",
+	{"arm64", 450}: "set_mempolicy_home_node",
+	{"arm64", 451}: "cachestat",
+	{"arm64", 452}: "fchmodat2",
+	{"arm64", 453}: "map_shadow_stack",
+	{"arm64", 454}: "futex_wake",
+	{"arm64", 455}: "futex_wait",
+	{"arm64", 456}: "futex_requeue",
+	{"arm64", 457}: "statmount",
+	{"arm64", 458}: "listmount",
+	{"arm64", 459}: "lsm_get_self_attr",
+	{"arm64", 460}: "lsm_set_self_attr",
+	{"arm64", 461}: "lsm_list_modules",
+	{"arm64", 462}: "syscalls",
```

Comment:
There is no such "syscalls" syscall :)

IMO ID 462 for both arm64 and amd64 should be related to "mseal"

Hopefully it's the last time we'll have to store syscalls ID lists somewhere :')